### PR TITLE
UIEH-1175: eHoldings - Removing Impagination dependency.

### PR DESCRIPTION
**Pull request by BogdanDenis**
## Description
The aim of this PR is to remove `impagination` dependency from `ui-eholdings` by replacing it with a custom component that will replicate the behaviour.
Added `<ImpaginationReplacement>` component that provides the same API as `<Impagination>`. Still we need to refactor `<QueryList>` and `<QuerySearchList>` components and combine them into one, and move `<ImpaginationReplacement>` logic into a hook. But this needs to be additionally discussed

### CODE REVIEW

File: src/components/scroll-view/scroll-view.js [hunk 1]
```diff
@@ -181,36 +160,16 @@ export default class ScrollView extends Component {
...
-        : PAGE_SIZE;
-    }
-
-    // list height should be at least enough for the offset
-    if (listHeight === 0) {
-      listHeight = offset + visibleItems;
-    }
-
-    listHeight *= itemHeight;
+    const listHeight = (length || items.length) * itemHeight;
```
Comment (Vladyslav-Velytskyi) [inline src/components/scroll-view/scroll-view.js:166]: What if `length` will be more then 100?  Isn't list will have empty space?

File: src/components/scroll-view/scroll-view.js [hunk 2]
```diff
@@ -181,36 +160,16 @@ export default class ScrollView extends Component {
...
 
     return (
       <div
         ref={(n) => { this.$list = n; }}
         className={cx('list', { locked: !scrollable })}
-        onScroll={this.handleScroll}
+        // onScroll={this.handleScroll}
```
Comment (mariiaschv) [inline src/components/scroll-view/scroll-view.js:172]: Do we need this comment here?

File: src/components/scroll-view/scroll-view.js (hunk 1 above)
Comment (BogdanDenis) [reply src/components/scroll-view/scroll-view.js]: This prop wasn't actually passed to the component from anywhere so I removed it

Comment (Vladyslav-Velytskyi) [reply src/components/scroll-view/scroll-view.js]: Ok, but if `items.length` will be more then 100?

**Review by Vladyslav-Velytskyi — APPROVED** [review]

**Review by mariiaschv — APPROVED** [review]

**Pull request merged**